src/renderer/pages/Operations/components/OperationSignatories.tsx

```diff
@@ -0,0 +1,171 @@
+import { useEffect, useState } from 'react';
+import { useUnit } from 'effector-react';
+
+import { BodyText, Button, CaptionText, FootnoteText, Icon, SmallTitleText } from '@renderer/shared/ui';
+import { AddressWithName, WalletIcon, walletModel } from '@renderer/entities/wallet';
+import { getSignatoryName } from '@renderer/pages/Operations/common/utils';
+import { AccountId, MultisigAccount, Signatory, Wallet } from '@renderer/shared/core';
+import { MultisigEvent, MultisigTransaction, SigningStatus } from '@renderer/entities/transaction';
+import { ExtendedChain } from '@renderer/entities/network';
+import { useI18n } from '@renderer/app/providers';
+import { useToggle } from '@renderer/shared/lib/hooks';
+import { nonNullable } from '@renderer/shared/lib/utils';
+import { contactModel } from '@renderer/entities/contact';
+import LogModal from './LogModal';
+import { useMultisigEvent } from '@renderer/entities/multisig';
+import { SignatoryCard } from '@renderer/entities/signatory';
+
+type WalletSignatory = Signatory & { wallet: Wallet };
+
+type Props = {
+  tx: MultisigTransaction;
+  connection: ExtendedChain;
+  account: MultisigAccount;
+};
+
+export const OperationSignatories = ({ tx, connection, account }: Props) => {
+  const { t } = useI18n();
+  const { getLiveTxEvents } = useMultisigEvent({});
+
+  const { signatories, accountId, chainId, callHash, blockCreated, indexCreated } = tx;
+  const events = getLiveTxEvents(accountId, chainId, callHash, blockCreated, indexCreated);
+
+  const contacts = useUnit(contactModel.$contacts);
+  const accounts = useUnit(walletModel.$accounts);
+  const walletsMap = new Map(useUnit(walletModel.$wallets).map((w) => [w.id, w]));
```

Comment:
you can use [useStoreMap](https://effector.dev/docs/api/effector-react/usestoremap/), but I am not sure what way is better

Reply:
`useStoreMap`  description in docs says it's common use is to subscribe to changes in selected part of store only and that doesn't look like our case so I think just `new Map` constructor is fine